data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/accumulator/BufferTypeOptions.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink.accumulator;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Defines all the buffer types enumerations.
+ */
+public enum BufferTypeOptions {
+
+    INMEMORY("in_memory", new InMemoryBuffer()),

```

Comment:
These should be `IN_MEMORY` and `LOCAL_FILE`. You can use your IDE to rename these easily.

Reply:
Followed the same notation (lower-case) similar to s3-source CompressionOption.